include/drivers/pwm.h

```diff
@@ -208,12 +208,6 @@ static inline int z_impl_pwm_pin_set_cycles(const struct device *dev,
  * @retval -EIO if IO error occurred while configuring
  * @retval -EBUSY if PWM capture is already in progress
  */
-static inline int pwm_pin_configure_capture(const struct device *dev,
```

Comment:
But doesn't this remove the anchor to which the documentation attaches, so it won't be documented?

Reply:
I think if the prototype removes `static inline` it might work.  I'm testing it in #31265 as well; see https://github.com/zephyrproject-rtos/zephyr/compare/e3d115573a9389e965c660693b9b0ce07cfc3301..57e3821567f4fe685a5b865269254007198d2db2

Reply:
@pabigot No, the documentation build still gets the correct result. I already tried several permutations of `inline`, `static`, `__unused` etc. without any luck.

Reply:
Ah: that's because you have `CONFIG_PWM_CAPTURE` in the doxygen predefines.  OK.